apps/rush-lib/src/cli/taskRunner/ProjectTask.ts

```diff
@@ -42,6 +42,10 @@ export default class ProjectTask implements ITaskDefinition {
     return this._rushProject.packageName;
   }
 
+  public get logName(): string {
+    return `${this.name} (${this._rushProject.projectRelativeFolder})`;
```

Comment:
>`${this.name} (${this._rushProject.projectRelativeFolder})`; [](start = 11, length = 60)

This formatting is pretty messy:

```
6 of 31: [@microsoft/gulp-core-build-karma (core-build/gulp-core-build-karma)] completed successfully in 6.96 seconds
7 of 31: [@microsoft/gulp-core-build-serve (core-build/gulp-core-build-serve)] completed successfully in 7.40 seconds
8 of 31: [@microsoft/api-extractor (apps/api-extractor)] completed successfully in 15.31 seconds
```

We should chat about the design.

CC @nickpape-msft

Reply:
Here's another possibility (are scopes interesting at all?):

```
6 of 31: [core-build/gulp-core-build-karma] completed successfully in 6.96 seconds
7 of 31: [core-build/gulp-core-build-serve] completed successfully in 7.40 seconds
8 of 31: [apps/api-extractor] completed successfully in 15.31 seconds
```


---
In reply to: [184757715](https://github.com/Microsoft/web-build-tools/pull/627#discussion_r184757715) [](ancestors = 184757715)